roles/openshift_node/tasks/main.yml

```diff
@@ -44,13 +44,6 @@
 - name: include node installer
   include_tasks: install.yml
 
-- name: Restart cri-o
-  systemd:
-    name: cri-o
-    enabled: yes
-    state: restarted
-  when: openshift_use_crio
-
```

Comment:
this was introduced by 39cf5084f18e2e0adca46b925660a6f2c38d227c from @ashcrow.  Is it still needed?  Until we can assure it is not needed I think we should leave this part when CRI-O is installed

Reply:
I have to admit I don't know if the SDN code changed. As @giuseppe noted, it was added because it needed to be restarted after the ``sdn-ovs`` was installed. It's possible that ``sdn-ovs`` is installed in a different area now.